src/experts/TimeInput.js

```diff
@@ -53,7 +50,7 @@
 			$.proxy( this._onRotatorChange, this ),
 			function() {
 				var value = self.viewState().value();
-				return value && value.getValue().calendar();
+				return value && value.getOption( 'calendarModel' );
```

Comment:
I was referring to this line. I fixed it in my commit, see https://github.com/wmde/ValueView/commit/929a80853a03c4a83e1d8b23ed5a1ca4454753c6#diff-49060d53e8ad9e0d74b8e4ea672589a5L53.